tests/JobCreatorTest.php

```diff
@@ -1022,7 +1010,7 @@ public function testComposerInstall(
         $yml = implode("\n", [
             str_replace('composer_install: false', 'composer_install: ' . $composerInstall, $this->getGenericYml()),
             <<<EOT
-            github_repository: 'silverstripe/installer'
+            github_repository: 'silverstripe/fake-module'
```

Comment:
Note that `silverstripe/installer` was incorrect here, and all the tests for this are assuming therefore that this is effectively some random repo in the silverstripe org. I felt it's best to preserve those tests as they are and explicitly indicate that this is just a random repo.

The alternative is to rename this to `silverstripe/silverstripe-installer` here but that fundamentally changes what's being tested. I think it's better that this works for any-old-random-repo rather than that it specifically works for installer.